covid19/printer.py

```diff
@@ -34,18 +34,10 @@ def get_in_info(self, message):
         return Rangebi.OKBLUE + str(message) + Rangebi.ENDC
 
 
-def new_lines(count=1):
-    i = 0
-    while i < count:
-        print("")
-        i += 1
-
-
-def get_tab(count=1):
-    i = 0
-    while i < count:
-        print("\t", end="")
-        i += 1
+new_lines = lambda count=1: print(end="\n"*count)
+
+
+get_tab = lambda count=1: print("\t"*count, end="")
```

Comment:
Can we directly call this variables from outside like functions ?

Reply:
Yes, we can. They are lambda functions, just like arrow functions in javascript.
you can test them in python shell